chia/consensus/default_constants.py

```diff
@@ -62,7 +62,7 @@
     "INITIAL_FREEZE_PERIOD": 42 * 4608,
     "NETWORK_TYPE": 0,
     "MAX_GENERATOR_SIZE": 1000000,
-    "MAX_GENERATOR_REF_LIST_SIZE": 10000,  # Number of references allowed in the block generator ref list
+    "MAX_GENERATOR_REF_LIST_SIZE": 10000,  # Number of references allowed in the block generator ref list -- seems high
```

Comment:
It seems that `MAX_GENERATOR_REF_LIST_SIZE` should be <= max number of transactions supported in a block, if we enforce that limit explicitly.

Reply:
We won't have explicit max tx limit, but this should be set to a more reasonable number.

Reply:
A "max number of transactions" isn't really a thing.